src/leveled_penciller.erl

```diff
@@ -1150,22 +1153,33 @@ handle_cast({fetch_levelzero, Slot, ReturnFun}, State) ->
     ReturnFun(lists:nth(Slot, State#state.levelzero_cache)),
     {noreply, State};
 handle_cast({log_level, LogLevel}, State) ->
-    PC = State#state.clerk,
-    ok = leveled_pclerk:clerk_loglevel(PC, LogLevel),
-    ok = leveled_log:set_loglevel(LogLevel),
+    case State#state.clerk of
```

Comment:
This code is repeated so often that it almost feels worth to add it to leveled_utils as
```
log_if_pid(undefined, _) -> ok;
log_if_pid(Pid, Fun) when is_pid(Pid) -> Fun().
```
And call it in different forms as:
`log_if_pid(State#state.clerk, fun()  -> leveled_pclerk:clerk_loglevel(PC, LogLevel) end)`

Matter of taste

Reply:
Repeated code now in function